spec/spec_helper.rb

```diff
@@ -11,7 +11,23 @@
 # a separate helper file that requires the additional dependencies and performs
 # the additional setup, and require it from the spec files that actually need
 # it.
-#
+
+if ENV.key? "COVERAGE"
```

Comment:
Does this just check if the key is present? Or is it also checking the truthiness of the value? (this is my lack of familiarity with Ruby showing through...)

I ask because it looks like the former and I'd be more comfortable with the latter.